experiments/random_layout_generation/random_layout_generation.cpp

```diff
@@ -0,0 +1,62 @@
+//
+// Created by Jan Drewniok on 04.05.23.
+//
+
+#include <fiction/algorithms/simulation/sidb/random_layout_generator.hpp>
+#include <fiction/io/read_sqd_layout.hpp>
+#include <fiction/types.hpp>
+
+#include <filesystem>
+#include <iomanip>
+#include <iostream>
+#include <string>
+
+using namespace fiction;
+
+int main()  // NOLINT
+{
+    // This script generates random layouts as .sqd file.
+    using cell_level_layout = cell_level_layout<sidb_technology, clocked_layout<cartesian_layout<cube::coord_t>>>;
+    const std::string     folder_name = "random_layouts_test/";
+    std::filesystem::path folder_path(EXPERIMENTS_PATH);
+    folder_path /= folder_name;
+
+    const bool result = std::filesystem::create_directory(folder_path);
```

Comment:
Not a very descriptive variable name. Also, this definition can be moved into the `if` condition